### PR TITLE
Shaloo/ar 4200 simple api usage

### PR DESCRIPTION
**Pull request by shaloo**
# Pull Request Template

Resolves or continues [AR-4200](https://team-1624093970686.atlassian.net/browse/AR-4200).

## Blocking dependencies

NA

## Changes

Usage.md has latest storage APIs that we recommend to the users.
usage_v2.md has v0.2 storage APIs (Access, Uploader and downloader objects) for reference and backwards compatibility until deprecated.

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -1,175 +1,181 @@
...
+4. [Access Control](#access-control)
+5. [Storage Usage Metrics](#storage-usage-metrics)
+6. [Download File by DID](#download-file-by-did)
+7. [Private NFT](#private-nft)
+8. [Managing Network and Wallet Account Changes](#managing-network-and-wallet-account-changes)
+
 ## Prerequisites
 
-To use the Storage SDK, you need to ensure that you have access to a blockchain `provider`.  This is required for signing blockchain transactions that power data access control in the Arcana protocol.
+Arcana Storage SDK provides data privacy and access control through blockchain transactions powered by the Arcana Network. To use the Storage SDK, you need to ensure that you have access to a Web3 wallet `provider`.  A provider is required for signing blockchain transactions.
```
Comment (mmjee) [inline usage.md:16]: Provider here is not referencing a piece of code. Code blocks are not appropriate.

File: usage.md [hunk 2]
```diff
@@ -1,175 +1,181 @@
...
-
-```typescript
-const downloader = await arcanaInstance.getDownloader();
-// did: Specify the DID of file that you'd like to download
-await downloader.download(did);
+2. Import `StorageProvider` from the Storage SDK package in the dApp. Call `init` method of `StorageProvider` and specify the Web3 wallet `provider` and the `appId` as input parameters. **Note:** Get the provider via the Auth SDK or third-party supported wallet. You can copy the appId from the [Arcana Developer Dashboard](https://docs.beta.arcana.network/docs/config_dapp) after registering your dApp
+3. Use `StorageProvider` to invoke:
+   - `upload` method to push file data into the Arcana Store. **Note:** Save file DID returned by upload.
+   - `download` method with file DID as input to retrieve a file from the Arcana Store.
+4. Use `StorageProvider.files` to invoke:
```
Comment (mmjee) [inline usage.md:31]: “Invoke delete” doesn't follow, I recommend dropping the “invoke”.

File: usage.md (hunk 1 above)
Comment (shaloo) [inline usage.md:16]:
```suggestion
Arcana Storage SDK provides data privacy and access control through blockchain transactions powered by the Arcana Network. To use the Storage SDK, you need to ensure that you have access to a Web3 wallet provider. A provider is required for signing blockchain transactions.

Comment (shaloo) [reply usage.md]: removed code block.

**Review by mmjee — APPROVED** [review]
Looks fine now, you'll have to rebase to `dev` though.

**Review by SauravKanchan — APPROVED** [review]

**Pull request merged**